gef.py

```diff
@@ -806,6 +867,28 @@ def heap_addr(self):
         _addr = int(self) + self.struct_size
         return malloc_align_address(_addr)
 
+    def get_heap_infos(self):
```

Comment:
nit: The word `infos` is awkward. Maybe we should name it something like `get_heap_info_list`